dialoguekit/connector/dialogue_connector.py

```diff
@@ -115,6 +116,16 @@ def register_agent_utterance(
         else:
             self._user.receive_utterance(annotated_utterance)
 
+    def register_user_feedback(
+        self, utterance_feedback: UtteranceFeedback
```

Comment:
Do we expect the caller to create an UtteranceFeedback object? Wouldn't it make more sense to take utterance_id and BinaryFeedback as arguments?

Reply:
Done